lib/services/product_brutto_price.rb

```diff
@@ -0,0 +1,8 @@
+module Store
+  class ProductBruttoPrice
+    def call(basket_id, product_id)
+      product = FetchProductFromBasket.new.call(basket_id, product_id)
+      product.quantity * product.price * (1 + product.vat)
```

Comment:
Once you put extra space around add and multiply (`*`, `+`) and once you skip it `price*(1+vat)*quantity`